### PR TITLE
Add base 3

### PR DESCRIPTION
**Pull request by kleberbaum**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently README doesn't serve it's purpose. #32 

**What is the new behavior (if this is a feature change)?**
- Now README serves it's purpose. #32 

**Other information**:
- Ref: https://help.github.com/en/articles/about-readmes

### CODE REVIEW

**Review by kleberbaum — COMMENTED** [review]
all peta clean and tidy ^^

File: README.md [hunk 1]
```diff
@@ -1,5 +1,173 @@
...
+
+To tail the logs from the Docker containers in realtime, run:
+
+```bash
+docker-compose logs -f
+```
+
+## [](#setup-with-python-virtual-environment)Setup with Python Virtual Environment
+
+You can start a Django project from this template without setting up Docker and simply use a virtual environment, which is the [recommended installation approach](https://docs.python.org/3/library/venv.html) for all Python projects itself.
```
Comment (Aichnerc) [inline README.md]: The max. line length is 119 characters.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,173 @@
...
+### Installation
+
+With [PIP](https://github.com/pypa/pip) installed, run:
+
+    git clone https://github.com/aichner/Django-Template.git
+    cd Django-Template
+    python --version
+    python -m pip --version
+
+Confirm that this is showing a compatible version of Python 3.x. If not, and you have multiple versions of Python installed on your system, you may need to specify the appropriate version when creating the venv:
```
Comment (Aichnerc) [inline README.md]: The max. line length is 119 characters.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,173 @@
...
+
+python3 -m venv /path/to/new/virtual/environment
+
+Once a virtual environment has been created, it can be “activated” using a script in the virtual environment’s binary directory. The invocation of the script is platform-specific (<venv> must be replaced by the path of the directory containing the virtual environment):
```
Comment (Aichnerc) [inline README.md]: The max. line length is 119 characters.

**Review by Aichnerc — CHANGES_REQUESTED** [review]
Please fix the line length. Then it should be ok to merge.

File: README.md (hunk 1 above)
Comment (kleberbaum) [reply README.md]: thx ^^

File: README.md (hunk 2 above)
Comment (kleberbaum) [reply README.md]: thx ^^

File: README.md (hunk 3 above)
Comment (kleberbaum) [reply README.md]: thx ^^

**Review by Aichnerc — APPROVED** [review]
Looks nya =^-^=

**Pull request merged**